### PR TITLE
Fix plugin url for single-host mode

### PR DESCRIPTION
**Pull request by tolusha**
### What does this PR do?
Fix plugin url for single-host mode

### What issues does this PR fix or reference?
https://github.com/eclipse/che/issues/17749

HAPPY_PATH_CHANNEL=stable

### CODE REVIEW

File: extensions/eclipse-che-theia-plugin-ext/src/node/che-plugin-service.ts [hunk 1]
```diff
@@ -329,7 +329,20 @@ export class ChePluginServiceImpl implements ChePluginService {
...
+                    // ends with `v3/plugins`, but ${plugin.links.self} starts with `/v3/plugins/${plugin.id}.
+                    // See https://github.com/eclipse/che-plugin-registry/blob/master/build/scripts/index.sh#L27
+                    // So the correct plugin url for both cases will be plugin registry url + plugin id.
+                    if (registry.uri.endsWith('/')) {
+                        return `${registry.uri}${plugin.id}/`;
+                    } else {
+                        return `${registry.uri}/${plugin.id}/`;
+                    }
+                }
+                const uri = new URI();
```
Comment (azatsarynnyy) [inline extensions/eclipse-che-theia-plugin-ext/src/node/che-plugin-service.ts:345]: `new URI()` ?
not `new URI(registry.uri)` ?

Comment (tolusha) [reply extensions/eclipse-che-theia-plugin-ext/src/node/che-plugin-service.ts]: `new URI(registry.uri)`
Fixed

**Review by azatsarynnyy — APPROVED** [review]
@tolusha please, merge it if you have checked it for both: single-/multi-host modes.
Thanks!

**Comment by tolusha** [comment]
@azatsarynnyy 
sure, I tested for both multi-host and single-host modes

**Pull request merged**